getx_demo/lib/app/modules/transactions/controllers/transactions_controller.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+
+import 'package:get/get.dart';
+
+import 'package:getx_demo/app/data/model/account.dart';
+import 'package:getx_demo/app/data/repository/account_repository.dart';
+import 'package:getx_demo/app/routes/app_pages.dart';
+
+class TransactionsController extends GetxController {
+  final accountRepository = Get.find<AccountRepository>();
+
+  Account get account => accountRepository.getMyAccount();
+
+  String get decimalPart =>
+      ((account.balance.value - account.balance.value.truncate()) * 100)
+                  .truncate() ==
+              0
+          ? '00'
+          : ((account.balance.value - account.balance.value.truncate()) * 100)
+              .truncate()
+              .toString();
+
+  loanButtonPress(BuildContext context) {
+    if (account.loanDecision.value == LoanDecisionRules.uninitiated) {
+      Get.toNamed(Routes.LOAN_APPLICATION);
+    } else {
+      showDialog(
+        context: context,
```

Comment:
```suggestion
        context: Get.context!,
```
For dialogs you don't need to keep track of context. Just use GetXs global one. Don't pass context as a parameter to the loanButtonPress function